app/models/affiliate.rb

```diff
@@ -0,0 +1,18 @@
+class Affiliate < ApplicationRecord
+
+  VALID_EMAIL_REGEX = /\A[\w+\-.]+@[a-z\d\-.]+\.[a-z]+\z/i
+
+  has_many :coordinators
+  has_many :tutors
+
+  has_many :matches
```

Comment:
I don't think it's necessary. We could still get that information through the affiliate's tutors and students. But hey, either way!